guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2937 @@
+{
```

Comment:
<ul><li><code>data_collections</code> and <code>analysis_variables </code> -&gt; remove plural form from code styling</li><li>...be found in <code>Enriching Study Areas</code> section. -&gt; use bold instead of code style</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>